<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="regimentâ€™s-MacBook-Pro" ignored="0" tests="1" failures="1" timestamp="2022-01-10T01:49:01 IST" time="8.641" errors="0">
  <testcase name="searchAndValidateGitRepo" time="8.426" classname="gitHubTest">
    <failure type="java.lang.AssertionError" message="Repository does not exist expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Repository does not exist expected [false] but found [true]
at gitHubTest.searchAndValidateGitRepo(gitHubTest.java:36)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- searchAndValidateGitRepo -->
</testsuite>
